File: Scripts/volatile_and_force_compiler_task/concurrent-computing/python/concurrent-computing.py
from __future__ import print_function
from multiprocessing import Pool

def executeTask(i):
  	p = Pool()
        p.map(print,'Enjoy Rosetta Code'.split())
	return i


def main():

    r = 1
    for i in range(1000000):
	r = executeTask(i)
if __name__=="__main__":
    main()

File: Scripts/volatile_and_force_compiler_task/concurrent-computing/go/concurrent-computing.go
package main

import (
    "fmt"
    "math/rand"
    "time"
)

func executeTask(i int) int {

    words := []string{"Enjoy", "Rosetta", "Code"}
    rand.Seed(time.Now().UnixNano())
    q := make(chan string)
    for _, w := range words {
        go func(w string) {
            time.Sleep(time.Duration(rand.Int63n(1)))
            q <- w
        }(w)
    }
    for j := 0; j < len(words); j++ {
	fmt.Println(i, <-q)
    }
return i
}

func main() {
    r := int(1)

    for j := 0; j < 1000000; j++ {
	r = executeTask(j + r)
}
}
